### ISSUE TITLE
Windows 10 Docker processes consuming high CPU..

### ISSUE THREAD

**Issue by guanxine** [docker/for-win#4773]
- [ ] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID:1DB7A834-1336-4391-97B2-C5353975B824/20190921044449

### Information

  - Windows Version: win10 18362.356
  - Docker Desktop Version: 2.1.0.1(37199)

### Steps to reproduce the behavior

  1. When my computer starts.. cup high
  2. ...

**Comment by jwhipp** [comment]
I ran into this again today but I didn't notice the CPU usage initially, I attempted to run docker build and received this error: 

Error response from daemon: mkdir /var/lib/docker/tmp/docker-builder723595540: read-only file system".   

Upon looking at the task manager I see the vmmem cranking the CPU.   I'd guess that the VM fails to boot correctly and is in a bad state.

As always, asking docker to restart (and thus bouncing the VM) solves the issue.
